src/main/java/seedu/address/model/statistic/GenderStatistic.java

```diff
@@ -0,0 +1,62 @@
+package seedu.address.model.statistic;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+
+/**
+ * Represents a GenderStatistic in the address book.
+ * Guarantees: immutable
+ */
+public class GenderStatistic {
+
+    private final int numberOfMales;
+    private final int numberOfFemales;
+    private final int numberOfOthers;
+    private final int total;
+
+    /**
+     * Constructs a {@code GenderStatistic}.
+     *
+     * @param numberOfMales Number of males
+     * @param numberOfFemales Number of females
+     * @param numberOfOthers Number of others
+     */
+    public GenderStatistic(int numberOfMales, int numberOfFemales, int numberOfOthers) {
+        requireAllNonNull(numberOfMales, numberOfFemales, numberOfOthers);
+        this.numberOfMales = numberOfMales;
+        this.numberOfFemales = numberOfFemales;
+        this.numberOfOthers = numberOfOthers;
+        this.total = numberOfMales + numberOfFemales + numberOfOthers;
+    }
+
+    private float computePercentage(int amount) {
+        return (float) amount / total * 100;
+    }
+
+    @Override
+    public boolean equals(Object other) {
+        if (other == this) {
+            return true;
+        }
+        if (!(other instanceof GenderStatistic)) { //this handles null as well.
+            return false;
+        }
+
+        GenderStatistic otherGenderStats = (GenderStatistic) other;
+
+        return this.numberOfMales == otherGenderStats.numberOfMales
+                && this.numberOfFemales == otherGenderStats.numberOfFemales
+                && this.numberOfOthers == otherGenderStats.numberOfOthers;
+    }
+
+    /**
+     * Format state as text for viewing.
```

Comment:
not a major issue for now but perhaps we could explore how to display these stats as diagrams in the next week